test/MessageTraitTest.php

```diff
@@ -305,31 +305,6 @@ public function testNumericHeaderValues()
         ];
     }
 
-    /**
-     * @dataProvider testNumericHeaderValues
-     * @group 99
-     */
-    public function testFilterHeadersShouldAllowIntegersAndFloats($value)
```

Comment:
I have tried, but there is some exclusive cases. Look at `invalidHeaderValues`. There are int and float as invalid value. This removed tests uses `testNumericHeaderValues` where is also int and float but as valid value